csrc/quantization/gguf/common.cuh

```diff
@@ -0,0 +1,1254 @@
+#define QK_K 256
```

Comment:
Is this copied from somewhere? Would be good to link to original source to give a pointer as to how updates should be conducted.